$NetBSD: patch-build-aux_depcomp,v 1.1 2017/03/06 16:14:07 jperkin Exp $

Sometimes, the same source file is compiled for both PIC and non-PIC use.
Avoid using the same temporary file in this case as it will result in
very hard to debug race conditions for parallel builds otherwise.

--- build-aux/depcomp.orig	2016-12-31 10:44:32.000000000 +0000
+++ build-aux/depcomp
@@ -124,7 +124,7 @@ fi
 # Dependencies for sub/bar.o or sub/bar.obj go into sub/.deps/bar.Po.
 depfile=${depfile-`echo "$object" |
   sed 's|[^\\/]*$|'${DEPDIR-.deps}'/&|;s|\.\([^.]*\)$|.P\1|;s|Pobj$|Po|'`}
-tmpdepfile=${tmpdepfile-`echo "$depfile" | sed 's/\.\([^.]*\)$/.T\1/'`}
+tmpdepfile=${tmpdepfile-`echo "$depfile" | sed 's/\.\([^.]*\)$/.T\1/'`}.$$
 
 rm -f "$tmpdepfile"
 
